packages/components/fields/number-field/src/number-field.tsx

```diff
@@ -278,7 +280,36 @@ class NumberField extends Component<TNumberFieldProps, TNumberFieldState> {
             id={sequentialErrorsId}
             errors={this.props.errors}
             isVisible={hasError}
-            renderError={this.props.renderError}
+            renderError={(key: string, error?: boolean) => {
+              const element = this.props.renderError?.(key, error);
```

Comment:
@ragafus Do you have a bit of time for us (remeet) to elaborate as to why this change is needed?

Reply:
> @ragafus Do you have a bit of time for us (remeet) to elaborate as to why this change is needed?

Of course! I can explain my point of view as a "consumer" of `NumberField` component.

Reply:
Great! I will set up a meeting for us. 
Thanks again